dhctl/pkg/app/bootstrap.go

```diff
@@ -53,6 +57,24 @@ func DefineDeckhouseFlags(cmd *kingpin.CmdClause) {
 		DurationVar(&DeckhouseTimeout)
 }
 
+func DefinePostBootstrapScriptFlags(cmd *kingpin.CmdClause) {
+	cmd.Flag("post-bootstrap-script-path", `Path to bash (or another interpreted language which installed on master node) script which will execute after bootstrap resources.
+All output of the script will be logged with Info level with prefix 'Post-bootstrap script result:'.
+If you want save to state cache on key 'post-bootstrap-result' you need to out result with prefix 'Result of post-bootstrap script:' in one line.
+Experimental. This feature may be deleted in the future.`).
+		Envar(configEnvName("POST_BOOTSTRAP_SCRIPT_PATH")).
+		StringVar(&PostBootstrapScriptPath)
+
+	cmd.Flag("post-bootstrap-script-timeout", "Timeout to execute after bootstrap resources script. Experimental. This feature may be deleted in the future.").
+		Envar(configEnvName("POST_BOOTSTRAP_SCRIPT_TIMEOUT")).
+		Default(PostBootstrapScriptTimeout.String()).
+		DurationVar(&PostBootstrapScriptTimeout)
+
+	cmd.Flag("post-bootstrap-script-exit-if-failed", "Exit with error if post bootstrap script was failed. Experimental. This feature may be deleted in the future.").
```

Comment:
I have a concern about this option. Should out soft think about it, or should the user write a script the way it will skip errors?